frontend_app_admin_workspace_user/i18next.scanner/en/translation.json

```diff
@@ -28,6 +28,7 @@
   "Error while deleting shared space": "Error while deleting shared space",
   "You can't remove yourself from Administrator": "You can't remove yourself from Administrator",
   "You can't deactivate your own account": "You can't deactivate your own account",
+  "Your username is incorrect, the allowed characters are azAZ09-_": "Your username is incorrect, the allowed characters are azAZ09-_",
```

Comment:
You probably should put "azAZ09-_" in a constant and use a format here so we don't have to change all the translation strings, should we change the allowed characters

Reply:
Changed